service/grpc/codelingo/codelingo.pb.go

```diff
@@ -258,13 +243,188 @@ func (m *GenFact) GetFactName() string {
 	return ""
 }
 
-func (m *GenFact) GetProperties() map[string]string {
+func (m *GenFact) GetProperties() map[string]*GenProperty {
 	if m != nil {
 		return m.Properties
 	}
 	return nil
 }
 
+func (m *GenFact) GetChildren() []*GenFact {
+	if m != nil {
+		return m.Children
+	}
+	return nil
+}
+
+type GenProperty struct {
+	// Types that are valid to be assigned to Value:
+	//	*GenProperty_String_
+	//	*GenProperty_Int
+	//	*GenProperty_Float
+	//	*GenProperty_Bool
+	Value isGenProperty_Value `protobuf_oneof:"value"`
+}
+
+func (m *GenProperty) Reset()                    { *m = GenProperty{} }
+func (m *GenProperty) String() string            { return proto.CompactTextString(m) }
+func (*GenProperty) ProtoMessage()               {}
+func (*GenProperty) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{6} }
+
+type isGenProperty_Value interface {
+	isGenProperty_Value()
+}
+
+type GenProperty_String_ struct {
+	String_ string `protobuf:"bytes,1,opt,name=string,oneof"`
+}
+type GenProperty_Int struct {
+	Int int64 `protobuf:"varint,2,opt,name=int,oneof"`
+}
+type GenProperty_Float struct {
```

Comment:
don't use underscores in Go names; type GenProperty_Float should be GenPropertyFloat